ra2ce/runners/direct_analysis_runner.py

```diff
@@ -48,11 +52,72 @@ def can_run(ra2ce_input: ConfigWrapper) -> bool:
         _network_config = ra2ce_input.network_config.config_data
         if not _network_config.hazard or not _network_config.hazard.hazard_map:
             logging.error(
-                "Please define a hazardmap in your network.ini file. Unable to calculate direct damages."
+                "Please define a hazard map in your network.ini file. Unable to calculate direct damages."
             )
             return False
         return True
 
+    @staticmethod
+    def _get_result_link_based(analysis: AnalysisDirectProtocol, analysis_config: AnalysisConfigWrapper,
+                               result_segment_based: gpd.GeoDataFrame) -> gpd.GeoDataFrame:
+        # Step 00: define parameters
+        base_graph_hazard_graph = analysis_config.graph_files.base_graph_hazard.graph
+        damage_curve = analysis.analysis.damage_curve.name
```

Comment:
Instead of working with `.name` it's better to work with the enum itself throughout the code. That also works in a string as there's an override `__str__` in `ra2ce_enum_base`.